### DISCUSSION TITLE
Bert 不识别 'ー'

### DISCUSSION THREAD

**Discussion by ffftian** [fishaudio/Bert-VITS2#327] [Q&A]
![4HN5KYSK@4BHNUCE@B(CD43](https://github.com/fishaudio/Bert-VITS2/assets/43436493/4765e45d-104c-4946-89b0-2fe9d413e9cd)
我之前有训练vits和so-vits-svc的经验，在我尝试用一致的训练集时他对ー报错，我改成汉字的一还是无法解决问题，日语中拥有大量的ー。理论不应该错在这上。

**Comment by meimisaki** [comment]
I encountered the same problem. The Japanese-Extra branch doesn't handle it correctly in the `clean_text` function. This error will be triggered when we feed it with something like "〜". As a workaround you can simply filter it out from the dataset.
